guides/common/modules/ref_supported-clients.adoc

```diff
@@ -0,0 +1,61 @@
+[id="supported-clients_{context}"]
+= Supported Clients
+
+{Project} supports the following operating systems and architectures.
+
+Supported Host Operating Systems::
+The hosts can use the following OS versions:
+
+ifdef::orcharhino[]
+* Alma Linux
+* Amazon Linux
+endif::[]
+ifndef::satellite[]
+* CentOS
+endif::[]
+ifndef::orcharhino,satellite[]
+* CoreOS
+endif::[]
+ifndef::satellite[]
+* Debian
+ifndef::orcharhino[]
+* Fedora
+* FreeBSD
+* Junos
+* OpenSUSE
+endif::[]
+* Oracle Linux
+endif::[]
+* {EL}
+** 9.0 or later
+** 8.0 or later
+** 7.2 or later
+ifdef::satellite[]
+** 6 with the https://www.redhat.com/en/resources/els-datasheet[ELS Add-On]
+endif::[]
+ifndef::satellite[]
+* {SLES}
+ifndef::orcharhino[]
+* Solaris 8, 10
+endif::[]
+Ubuntu
+endif::[]
+
+Supported Host Architectures::
+The hosts can use the following architectures:
+
+{EL}:
+
+* i386
+* x86_64
+* s390x
+* ppc_64
+
+ifndef::satellite[]
+// WIP
+Architectures: Foreman x86_64/amd64
+orcharhino
+aarch64 for Debian10/11, SLES 15, Ubuntu 20.04/22.04;
+ppc64le for Debian 10 and SLES 15;
+and x86_64/amd64 for all OS.
```

Comment:
AFAIK for vanilla Foreman the architecture doesn't matter at all. There is no client tooling we supply so if the OS runs, it should work.

Reply:
But Foreman does supply some client tooling: https://yum.theforeman.org/client/latest/ and it looks to me that it's only available for x86_64.

Reply:
> @ekohl But Foreman does supply some client tooling: https://yum.theforeman.org/client/latest/

Yes, there is some client tooling. Most notably it's:

* `rubygem-foreman_scap_client` for OpenSCAP
* `katello-host-tools`, which is to upload package information to Katello
* `katello-host-tools-tracer`, which is to upload traces to Katello. They indicate that services or the whole machine needs to be restarted
* `foreman_ygg_worker` for REX pull mode

All of that is optional or unsupported for vanilla Foreman.

> Also, it looks to me that it's only available for x86_64 (guessing by the repo content). @maximiliankolb ?

You're right that today we only build it for x86_64, but that is mostly an issue on our package building process. We're in the process of migrating it to COPR which should make it possible to build for multiple architectures.